docs/modules/clientutils.rst

```diff
@@ -214,6 +214,25 @@ Retrieves boundaries for all DOM element matching the provided :ref:`selector <s
 
 It returns an array of objects each having four keys: ``top``, ``left``, ``width`` and ``height``.
 
+.. index:: Link text
+
+``getElementByLinkText()``
+-------------------------------------------------------------------------------
+
+**Signature:** ``getElementByLinkText(String text [, HTMLElement scope])``
+
+Retrieves a single DOM element matching a given :ref:`Link text <selectors>`.
+
+.. versionadded:: 1.1
+
+The ``scope`` argument allow to set the context for searching text on "a" tags::
```

Comment:
Why only `a` tags? What about `<span onclick="fn()">text</span>`, `<button onclick…>` and friends?